src/MoBi.Presentation/Presenter/Main/NotificationPresenter.cs

```diff
@@ -207,9 +191,24 @@ private void updateNotificationWith(IEnumerable<NotificationMessageDTO> notifica
 
       private bool notificationIsVisible(NotificationMessageDTO message)
       {
+         if (!_userSettings.ShowUnresolvedEndosomeMessagesForInitialConditions && isUnresolvedEndosomeForInitialConditionMessage(message))
+            return false;
+         
          return _userSettings.ShowPKSimObserverMessages || !isPKSimObserverMessage(message);
       }
 
+      private bool isUnresolvedEndosomeForInitialConditionMessage(NotificationMessageDTO message)
+      {
+         if (!(message.Object is InitialCondition initialCondition)) 
+            return false;
+
+         // Only applies to endosome containers. Don't ignore other unresolved containers
+         if (!string.Equals(initialCondition.ContainerPath.Last(), Endosome))
+            return false;
+
+         return message.NotificationMessage.Message.Equals(Validation.StartValueDefinedForContainerThatCannotBeResolved(message.ObjectName, initialCondition.ContainerPath));
```

Comment:
Testing the text here to see if this is the kind of message we should ignore. I don't know if it makes sense to have this test. Presumably there could be some other warning message for this container that we would incorrectly ignore if we don't somehow test for something specific to this notification.